mitiq/shadows/classical_postprocessing.py

```diff
@@ -373,10 +367,12 @@ def expectation_estimation_shadow(
                     # witch satisfy condition (1) and (2)
                     product = (1.0 / f_val) * product
             else:
-                product = 3 ** (target_support.count("1")) * product
+                product = 3 ** (len(target_locs)) * product
```

Comment:
I believe the problem with the bad results is from this line. It should be sum(target_locs) since this would be the number of 1s in the target_locs list of 0s and 1s. I'm testing locally to see if the results in the documentation are valid again.
@andreamari @Min-Li